src/homework07/Bankomat.java

```diff
@@ -0,0 +1,143 @@
+package homework07;
+
+public class Bankomat {
```

Comment:
Try to give English names to classes and variables
EmulatorATM
getTotalSum

Reply:
fix